### PR TITLE
Account settings

### PR DESCRIPTION
**Pull request by ixhbinphoenix**
Good fucking luck reviewing this

### CODE REVIEW

File: frontend/src/pages/home.astro [hunk 1]
```diff
@@ -1,6 +1,20 @@
 ---
 import PlanLayout from "../layouts/PlanLayout.astro";
 import PlanContainer from "../components/PlanContainer";
+import { checkSessionIdAstro } from "../api/frontmatter";
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```
Comment (Sxyntheon) [inline frontend/src/pages/home.astro:5]: And that

File: frontend/src/pages/login.astro [hunk 2]
```diff
@@ -1,6 +1,19 @@
 ---
 import LoginLayout from "../layouts/LoginLayout.astro";
 import LoginForm from "../components/LoginForm";
+import { checkSessionIdAstro } from "../api/frontmatter";
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"; //super unsafe code here btw
```
Comment (Sxyntheon) [inline frontend/src/pages/login.astro:5]: And that

File: frontend/src/pages/registrieren.astro [hunk 3]
```diff
@@ -1,6 +1,21 @@
 ---
 import LoginLayout from "../layouts/LoginLayout.astro";
 import RegisterForm from "../components/RegisterForm";
+import { checkSessionIdAstro } from "../api/frontmatter";
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"; //super unsafe code here btw
```
Comment (Sxyntheon) [inline frontend/src/pages/registrieren.astro:5]: And that

File: frontend/src/pages/reset-email/[id].astro [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+import ResetPasswordLayout from "../../layouts/ResetPasswordLayout.astro";
+import ResetEmail from "../../components/plan-components/settings-components/ResetEmail";
+import { checkUUID } from "../../api/frontmatter";
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```
Comment (Sxyntheon) [inline frontend/src/pages/reset-email/[id].astro:5]: And that

File: frontend/src/pages/reset-password/[id].astro [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+import ResetPasswordLayout from "../../layouts/ResetPasswordLayout.astro";
+import ResetPassword from "../../components/plan-components/settings-components/ResetPassword";
+import { checkUUID } from "../../api/frontmatter";
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```
Comment (Sxyntheon) [inline frontend/src/pages/reset-password/[id].astro:5]: And that

File: frontend/src/pages/verify/[id].astro [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+import ResetPasswordLayout from "../../layouts/ResetPasswordLayout.astro";
+import VerifiyAccount from "../../components/plan-components/settings-components/VerifyAccount";
+import { checkUUID } from "../../api/frontmatter";
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
```
Comment (Sxyntheon) [inline frontend/src/pages/verify/[id].astro:5]: And that

**Review by Sxyntheon — APPROVED** [review]
lgtm

File: frontend/src/api/theBackend.ts [hunk 7]
```diff
@@ -1,215 +1,279 @@
...
-      message: "Server connection failed"
-    };
+    return Promise.reject(error);
   }
 }
 export async function registerAccount(
   email: string,
-  hashedPassword: string,
+  password: string,
+
```
Comment (ixhbinphoenix) [inline frontend/src/api/theBackend.ts:72]: why

**Review by ixhbinphoenix — COMMENTED** [review]
Mach ma nen pnpm format oder so keine ahnung darf keine changes requesten ig

**Review by ixhbinphoenix — COMMENTED** [review]
lgtm

**Pull request merged**